cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -44,7 +44,7 @@ def vm_obj(request, provider, setup_provider, console_template):
 
     request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
 
-    vm_obj.create_on_provider(timeout=2400, find_in_cfme=True, allow_skip="default")
+    vm_obj.create_on_provider(request, timeout=2400, find_in_cfme=True, allow_skip="default")
```

Comment:
Do you want to also search for other usages of `create_on_provider` and add `request` where it can be added? What about fixtures and functions that do not currently depend on request?